cdci_data_analysis/analysis/instrument.py

```diff
@@ -270,8 +270,10 @@ def run_query(self, product_type,
         #  this was removed by 2f5b5dfb7e but turns out it is used by some plugins, see test_server_plugin_integral_all_sky
         self._current_par_dic=par_dic
 
-        # set pars values from the input parameters
-        query_out = self.set_pars_from_form(par_dic, verbose=verbose, sentry_client=sentry_client)
+        # # set pars values from the input parameters
+        # query_out = self.set_pars_from_form(par_dic, verbose=verbose, sentry_client=sentry_client)
```

Comment:
So this place is where the second unnecessary conversion occurred!

Reply:
Exactly, we didn't notice during the first refactoring. But now it is quite clear the "damage" it creates.